components/src/molecules/dropdown-list/DropdownList.html

```diff
@@ -1,19 +1,15 @@
-<div :class="['dropdown-list', {
-  'dropdown-list--inner': inner,
-  'dropdown-list--secondary': secondary,
-  'dropdown-list--dark': dark,
-  'dropdown-list@screen-m dropdown-list--is-open@screen-m': screenM
-  }]"
->
+<div :class="['dropdown-list', { 'dropdown-list--inner': nested }]">
```

Comment:
`dropdown-list--inner` class can be passed in story, I think we can leave here only `class="dropdown-list"`

Reply:
We can pass `dropdown-list--inner` in the story, if we want  `inner styles`, but here it's for nested dropdown. During using it we can't pass this modifirer just in root, because it will be used for all dropdown, we need it only for nested elements.